index.js

```diff
@@ -778,3 +791,7 @@ async function getThemeRepos(page = 1) {
 
     return filteredResults;
 }
+function getUnloadableManifests() {
```

Comment:
This function (and the session storage query from `getRepoManifests`) could be replaced by a single `getSessionStorageValue` function like the one from `index.js`. 

```js
/**
 * Get localStorage data (or fallback value), given a key
 * @param {string} key The localStorage key
 * @param {any} fallback Fallback value if the key is not found
 * @returns The data stored in localStorage, or the fallback value if not found
 */
const getLocalStorageDataFromKey = (key, fallback) => {
    const str = localStorage.getItem(key);
    if (!str) return fallback;

    const obj = JSON.parse(str);
    return obj;
};
```